stl/inc/ranges

```diff
@@ -2196,7 +2196,7 @@ namespace ranges {
 
         _NODISCARD constexpr auto begin() {
 #if _CONTAINER_DEBUG_LEVEL > 0
-            _STL_VERIFY(_Pred, "LWG-3490 forbids calling begin on a drop_while_view with no predicate");
+            _STL_VERIFY(_Pred, "[range.drop.while.view] forbids calling begin on a drop_while_view with no predicate");

```

Comment:
This citation needs a document number. (This will be hard to obtain until there's a new working draft.)

Reply:
We can wait for the new WP. Marking as blocked until then.

Reply:
The next Working Draft will be N4885 (https://github.com/cplusplus/draft/commit/bd2cc23e7cc72d5fc0445da1955dc4b46f7caa25. I'll unblock this PR, confident that we can come back and fix things up in the unlikely case that WG21 repeats the "same document number assigned to two different documents" fiasco.